ACE/debian/ace.dsc

```diff
@@ -1,10 +0,0 @@
-Format: 1.0
```

Comment:
Check ACE/bin/make_release.py, I think the script expects this file, also that python script has to be checked for all other file renames, it updates the content from 6.4.7 to 6.4.8 with the next release